src/utilities/effects.ts

```diff
@@ -2656,7 +2656,7 @@ const effects = {
     try {
       const data = await reqHasura(convertToQuery(gql.SUB_PLAN_TAGS), { planId }, user);
       const { tags } = data;
-      return tags;
+      return tags || [];
```

Comment:
do we know why `tags` isn't being set as an empty array from the query?

Reply:
Not entirely sure.. was wondering that myself. There's nothing in this path that explicitly checks for a null response and if so returns an empty array. Guess in a lot of other places we do null checks and/or check on the initial data and/or have flexibility in the type.

Reply:
Yeah, I just wanted to make sure that we explicitly handle the null case if we needed to. Usually if a query comes back as `null`, we throw an explicit error or return `null` so that a `null` case is handled in the UI. My worry is that returning `[]` by default might be hiding some weird state or something.

Reply:
Aha, was actually a real bug - we weren't unpacking the tags correctly out of the plan.

Reply:
nice catch!